packages/python-sdk/tests/test_session.py

```diff
@@ -15,3 +15,26 @@ async def test_create_multiple_sessions():
     session2 = await Session.create("Nodejs", api_key=E2B_API_KEY)
     await session.close()
     await session2.close()
+
+
+async def test_custom_cwd():
+    session = await Session.create("Nodejs", api_key=E2B_API_KEY, cwd="/code/app")
+
+    proc = await session.process.start("pwd")
+    output = await proc
+    assert output.stdout == "/code/app"
+
+    await session.filesystem.write("hello.txt", "Hello VM!")
```

Comment:
This creates a file in `/`. I am changing it in another PR, it will respect `cwd`. This will return error, if you try e.g. this `cat hello.txt`

Reply:
You can just add todo with E2B-184

Reply:
ooof, very good catch!
Changed the tests for now, as it's a little unintuitive, as it should be tested in the following PR
![Screen 2023-09-27 at 08 38 58](https://github.com/e2b-dev/e2b/assets/697301/05c2b908-a8f6-4be4-b44a-fb09c754b731)

Reply:
It's confusing also Copilot :D 
![Screen 2023-09-27 at 08 41 42](https://github.com/e2b-dev/e2b/assets/697301/f91ffce1-90aa-4414-82cc-07250f6554cd)